### PR TITLE
Bru2nii

### PR DESCRIPTION
**Pull request by TheChymera**
Modern file conversion utility for the Bruker format into the more popular NIfTI.

### CODE REVIEW

File: sci-biology/bru2nii/bru2nii-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+LICENSE="|| ( Artistic GPL-1+ )"
+SLOT="0"
+IUSE="gui"
+
+RDEPEND=""
+DEPEND="dev-lang/fpc
+	gui? ( dev-lang/lazarus )"
+
+src_compile() {
+	fpc -Tlinux Bru2.lpr
```
Comment (jlec) [inline sci-biology/bru2nii/bru2nii-9999.ebuild]: missing ```die```

File: sci-biology/bru2nii/bru2nii-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	use gui && lazbuild -B --lazarusdir="/usr/share/lazarus/" Bru2Nii.lpr
```
Comment (jlec) [inline sci-biology/bru2nii/bru2nii-9999.ebuild]: missing ```die```

File: sci-biology/bru2nii/bru2nii-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+src_install() {
+	dobin Bru2 || die "dobin failed"
```
Comment (jlec) [inline sci-biology/bru2nii/bru2nii-9999.ebuild]: dobin dies on its own, so you can drop it.

File: sci-biology/bru2nii/bru2nii-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+	use gui && dobin Bru2Nii || die "dobin failed"
```
Comment (jlec) [inline sci-biology/bru2nii/bru2nii-9999.ebuild]: same here.

**Comment by TheChymera** [comment]
@jlec bump?

File: sci-biology/bru2nii/bru2nii-9999.ebuild [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+SLOT="0"
+IUSE="gui"
+
+RDEPEND=""
+DEPEND="dev-lang/fpc
+	gui? ( dev-lang/lazarus )"
+
+src_compile() {
+	fpc -Tlinux Bru2.lpr || die
+	use gui && (lazbuild -B --lazarusdir="/usr/share/lazarus/" Bru2Nii.lpr || die)
```
Comment (jlec) [inline sci-biology/bru2nii/bru2nii-9999.ebuild]: Please check https://devmanual.gentoo.org/ebuild-writing/error-handling/index.html#die-and-subshells

You need to use an if then clause here.

**Comment by TheChymera** [comment]
@jlec done :)

File: sci-biology/bru2nii/bru2nii-9999.ebuild [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+SLOT="0"
+IUSE="gui"
+
+RDEPEND=""
+DEPEND="dev-lang/fpc
+	gui? ( dev-lang/lazarus )"
+
+src_compile() {
+	fpc -Tlinux Bru2.lpr || die
+	if [[ -f gui ]] ; then
```
Comment (jlec) [inline sci-biology/bru2nii/bru2nii-9999.ebuild]:
```ebuild
if use gui; then
...
```

**Comment by TheChymera** [comment]
@jlec  now?

**Pull request merged**